### PR TITLE
Request updated batman components for Azure

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: azure/requests.yaml [hunk 1]
```diff
@@ -1,20 +1,31 @@
 releases:
     - name: ">= 15.0.0"
       requests:
+        - name: app-operator
+          version: ">= 4.1.0"
+          issue: https://github.com/giantswarm/roadmap/issues/254
```
Comment (rossf7) [inline azure/requests.yaml:6]: With this we create an app-operator per workload cluster. This is new functionality so requesting for the next major release.

In the release CR releaseOperatorDeploy needs to be set to false.

File: azure/requests.yaml [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
         - name: cert-operator
           version: ">= 1.0.1"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: cluster-operator
-          version: ">= 0.24.2"
-          issue: https://github.com/giantswarm/giantswarm/issues/14205
+          version: ">= 0.25.1"
+          issue: https://github.com/giantswarm/roadmap/issues/254
```
Comment (rossf7) [inline azure/requests.yaml:12]: Related changes for creating an app-operator per workload cluster.

File: azure/requests.yaml [hunk 3]
```diff
@@ -1,20 +1,31 @@
...
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: "> 14.1.2"
+      requests:
+        - name: app-operator
+          version: ">= 3.2.1 < 4.0.0"
+          issue: https://github.com/giantswarm/giantswarm/issues/15920
```
Comment (rossf7) [inline azure/requests.yaml:20]: This includes a security fix for the status webhook ingress.

**Comment by rossf7** [comment]
Tested on gremlin.

**Review by whites11 — APPROVED** [review]

**Pull request merged**